karma-shim.js

```diff
@@ -10,13 +10,15 @@ global.$ = global.jQuery = require('jquery');
 
 import './settings';
 import './app/scripts/app';
+import './test/helpers/customMatchers';
```

Comment:
this file is really finicky about import order and call order.  Reordering all imports to the top breaks a good 10% of our tests.